imports/client/components/SplitPanePlus.tsx

```diff
@@ -1,92 +1,130 @@
-                        set size within the automatic collapse zone without triggering a collapse.
+  Props:
+    size: number
+      Size in pixels of the primary pane. Size has no effect during drag or if collapsed is other
+      than 0. If size is set to a value normally prohibited by minSize or maxSize or within any
+      autoCollapse range, the primary pane will still render at the requested size. If size is
+      larger than the container, the primary pane will fill the container. In no event does size
+      alone trigger collapse.
+    collapsed?: 0 | 1 | 2 = 0
+      Which pane is currently collapsed (reduced to size 0 with appropriate classes applied).
+      If 0, neither pane is collapsed and normal dragging behavior is present.
+    primary?: 'first' | 'second' = 'first'
+      Which pane the size property is applied to
+    split?: 'vertical' | 'horizontal' = 'vertical'
+      Orientation of the dividing line. 'vertical' is side-by-side and 'horizontal' is stacked.
+    allowResize?: boolean = true
+      True to enable user drag.
+    minSize?: number = 0
+      Minimum size of the primary pane. Respected by drag and resize.
+    maxSize?: number = 0
+      Maximum size of the primary pane. Respected by drag and resize. If zero or negative,
+      interpreted as minimum size of secondary pane.
+    autoCollapse1?, autoCollapse2?: number = 50
+      Size in pixels below which a pane is automatically collapsed. Respected by drag and resize.
+      Set negative to disable.
+    scaling?: 'absolute' | 'relative' = 'absolute'
+      If 'relative', primary and secondary panes will retain their proportions during resize.
+    step?: number
+      Step size when dragging
+    onPaneChanged?: (size: number, collapsed: 0 | 1 | 2, cause: 'drag' | 'resize') => void
+      Callback triggered to request a change to size or collapsed state. Drag events will call
+      this once, at the end of the event. Resize events will call this whenever the currently
+      implied condition does not match the existing props.
+    className?: string
+      Class names applied to the parent div, in addition to 'SplitPane' and 'Vertical' or
+      'Horizontal' (depending on split)
+    style?: React.CSSProperties
+      Styles applied to the parent div
+    paneClassName?, pane1ClassName?, pane2ClassName?: string
+      Class names applied to both, the first, and the second pane respectively, in addition to
+      'Pane', 'Pane1', and 'Pane2' as well as 'Vertical' or 'Horizontal' (depending on split)
+    paneStyle?, pane1Style?, pane2Style?: React.CSSProperties
+      Styles applied to both, the first, and the second pane respectively
+    resizerClassName?: string
+      Class names applied to the resizer, in addition to 'Resizer' as well as 'Vertical' or
+      'Horizontal' (depending on split)
+    resizerStyle?: React.CSSProperties
+      Styles applied to the resizer
```

Comment:
Nit, and we can follow up later, but it'd be nice to move these comments onto the prop interface definition, since the types already live there

Reply:
That's a good point, but I'm afraid it's getting pushed to next week...